spec/fixtures/llm/google_gemini/embed.json

```diff
@@ -0,0 +1,9 @@
+{
+  "embedding": {
+    "values": [
+      0.013168523,
```

Comment:
truncated the embedding values here because it looks like you also did that for the Vertex fixtures. Thinking that's ok?